data/qihoo360

```diff
@@ -1,15 +1,22 @@
 360.com
-360.cn
```

Comment:
why remove this domain

Reply:
Because `qihoo360` is included in `geolocation-cn`. Remove the `.cn` domain according to `tld-cn`.

Reply:
If so, how can I block qihoo360?

Reply:
OK, I see what you mean.